massa-protocol-worker/src/handlers/block_handler/cache.rs

```diff
@@ -47,6 +47,35 @@ impl BlockCache {
             max_known_blocks_by_peer,
         }
     }
+
+    pub fn update_cache(
+        &mut self,
+        peers_connected: HashSet<PeerId>,
+        max_known_blocks_by_peer: u32,
+    ) {
+        let peers: Vec<PeerId> = self
+            .max_known_blocks_by_peer
```

Comment:
max_known_blocks_by_peer is u32 no ?

Reply:
Fixed. I will fix my IDE also he was like "everythig is fine"

Reply:
me too since few days

Reply:
Compilation ended on my PC I fixed everything should be ok now